server/app/controllers/admin/AdminProgramController.java

```diff
@@ -151,15 +152,27 @@ public Result create(Request request) {
       return ok(newOneView.render(request, programData, message));
     }
 
-    return getSaveProgramDetailsRedirect(request, result.getResult().id());
+    return getSaveProgramDetailsRedirect(
+        request, result.getResult().id(), /* isInCreationFlow= */ true);
```

Comment:
suggestion (non-blocking): I'm channeling Bion in this suggestion, who would always recommend making params enums instead of booleans if possible since it makes them more extensible and then we don't have to include the param hints everywhere we invoke the method. I think in this case that might make sense since I could see us potentially having different flows in the future. You might be able to use the `ProgramEditStatus` enum for this (or maybe `Referer`?). I don't feel super strongly about this, but wanted to offer it as an alternative.

Reply:
Updated to use `ProgramEditStatus`!